spec/controllers/rooms_controller_spec.rb

```diff
@@ -173,6 +173,26 @@
       expect(response).to have_http_status(:forbidden)
     end
 
+    context 'the access code meeting option is enabled' do
+      let(:fake_room_settings_getter) { instance_double(RoomSettingsGetter) }
+      let!(:meeting_option) { create(:meeting_option, name: 'glViewerAccessCode') }
+      let!(:room_meeting_option) { create(:room_meeting_option, meeting_option:) }
```

Comment:
Why are these needed if you're stubbing the RoomSettingsGetter?

Reply:
The RoomSettingsGetter will return the settings values, but AFAIK, it doesn't actually create the RoomMeetingOption and MeetingOption? I will double check.

Reply:
You shouldn't need to - the `generate_access_code` method should create the RoomMeetingOption for you